data/human/culture republic.txt

```diff
@@ -0,0 +1,124 @@
+			`	Curious, you turn over your own mug and see that it is indeed at least five times what you would expect to pay for a mug, even a good one. Still, it is certainly a novelty...`
+			choice
+				`	(Buy the mug.)`
+					goto buy
+				`	(Put it back.)`
+			`	You roll your eyes and put it back. Some experiences really are universal.`
+				decline
+			label buy
+			`	You purchase the mug on an impulse. At least you can ditch your other one with a chip in it.`
+				decline
+
+
+mission "ZZCU Trash Fire Crops"
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`As you move about the spaceport you can't avoid the terrible smell. It's like a trash fire, but you can't see any sort of smoke.`
+			choice
+				`	(Ignore it.)`
+					goto ignore
+				`	(Ask around.)`
+			`	You ask one of the locals who doesn't look too busy. They sadly inform you that there's been a bumper banana crop, but unfortunately other wealthier worlds had a bumper crop at the same time, and there's nowhere to either sell or store a significant excess. Distressed farmers are burning the excess fruit to make fertilizer for the next season, the wind has carried the smell all over town.`
+				decline
+			label ignore
+			`	You leave the mysterious smell to the locals and continue on your business.`
+				decline
+
+
+mission "ZZCU Paradise Cleaner Incident"
+	minor
+	source
+		government "Republic"
+		attributes "paradise"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Visiting the Paradise spaceports is always an interesting experience for someone from abroad. Nowhere else is class such an ingrained component of daily life. You find yourself noticing people who seem to be invisible to most of those going about their business, but they never catch your eye. They always appear to be doing their utmost to avoid notice.`
+			`	You haven't noticed children that look poor here in the past, but on this occasion you spot one right before he knocks over a mop and bucket... right in front of a businessman and his family rolling their bags through the spaceport. You've never seen a mother go so deathly pale as the cleaner who scurried to fix things before the man had even drawn breath.`
+			`	A pair of security officers appear almost immediately, before you could intercede one way or the other, as the man and his family begin to complain. One officer provides assurances as the other supervises the cleaner, who quickly hurries her children out of harms way.`
```

Comment:
```suggestion
			`	A pair of security officers appear almost immediately, before you could intercede one way or the other, as the man and his family begin to complain. One officer provides assurances as the other supervises the cleaner, who quickly hurries her child out of harms way.`
```
She isn't really mentioned as having more than one child before, so it feels a little weird to bring it up here. You could also keep this as-is and add some mention of her having additional kids earlier, but that might be more work.